### PR TITLE
Switch classnames to rt-search__..

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

**Comment by friedjoff** [comment]
Nice! Can you also make the className configurable using props like in our other components. Custom className should also be applied to the theme classes.

**Comment by edellucien** [comment]
> Nice! Can you also make the className configurable using props like in our other components. Custom className should also be applied to the theme classes.

I see! Good idea

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -64,10 +71,28 @@ function Search({
     [apiKey, engines, setSuggestions],
   );
 
+  const theme = {
```
Comment (friedjoff) [inline src/components/Search/Search.js]: Maybe we should put this into a `useMemo` hook? Might be a small performance improvement.

Comment (edellucien) [reply src/components/Search/Search.js]: Yes it is better so :)

**Pull request merged**